dev-packages/browser-integration-tests/suites/tracing/request/fetch-with-request/test.ts

```diff
@@ -10,21 +10,19 @@ sentryTest(
       sentryTest.skip();
     }
 
-    await page.route('**/api/test/', async route => {
-      const req = route.request();
-      const headers = await req.allHeaders();
-
-      // headers.bar was set in fetch options (and should be sent)
-      expect(headers.bar).toBe('22');
-      // headers.foo was set in init request object (and should be ignored)
-      expect(headers.foo).toBeUndefined();
-
-      return route.fulfill({
-        status: 200,
-        body: 'ok',
-      });
-    });
-
-    await getLocalTestUrl({ testDir: __dirname });
+    const requestPromise = page.waitForRequest('http://example.com/api/test/');
```

Comment:
this test was previously missing a `page.goto` call, causing the assertions to not even be executed. Cleaned up the test to ensure that we now actually assert on the headers (and fail if they don't match)